kubefirst/gitops-template
# feat(aws-gitlab): add ability to create a physical cluster in aws gitlab

## Description


> This is broadly the same as #774, but for GitLab

## Related Issue(s)

Fixes #782 
Fixed #764 

## How to test


See #774 for the test commands as everything is identical from a Console point of view. The required command to create the cluster is:

```bash
../kubefirst/kubefirst aws create \
  --alerts-email <email> \
  --domain-name <domain> \
  --cluster-name <your-cluster-name> \
  --gitlab-group <gitlab-group> \
  --cloud-region <region> \
  --gitops-template-url https://github.com/mrsimonemmsorg/gitops-template.git \
  --gitops-template-branch sje/aws-gitlab-cluster \
  --git-provider gitlab
```

Note that `gitlab-group`, `gitops-template-branch` and `git-provider` are different to the GitHub command.
